include/GafferImage/AtomicFormatPlug.h

```diff
@@ -49,3 +49,14 @@ using AtomicFormatPlug = Gaffer::TypedPlug<GafferImage::Format>;
 IE_CORE_DECLAREPTR( AtomicFormatPlug );
 
 } // namespace GafferImage
+
+namespace Gaffer
+{
+
+template<>
+GafferImage::Format GafferImage::AtomicFormatPlug::getValue( const IECore::MurmurHash *precomputedHash ) const;
+
+template<>
+IECore::MurmurHash GafferImage::AtomicFormatPlug::hash() const;
+
+} // namespace Gaffer
```

Comment:
Github is now listing a missing newline at EOF here - I don't think we should be doing that, though the whitespace checker isn't complaining about it.

Reply:
Also fixed in https://github.com/GafferHQ/gaffer/pull/5362/commits/c7e4e53a7ac3598d4d614f8ae5d05121cc076b30.